generic_scripts/ids_feedforward_vchamber_effect.py

```diff
@@ -0,0 +1,207 @@
+"""Model of the vacuum chamber effect on oscillating field.
+
+Script that calculates the effect of the vacuum chamber on the performance of
+the feedforward system.
+
+The modelling of the transfer function of the vacuum chamber is based on the
+paper:
+
+Podobedov, B., Ecker, L., Harder D., Rakowsky G.,
+Eddy Current Shielding By Electrically Thick Vacuum Chambers
+Proceedings of PAC09, Vancouver, BC, Canada
+code: TH5PFP083, pp 3398-3400. Available at:
+https://accelconf.web.cern.ch/pac2009/papers/th5pfp083.pdf
```

Comment:
:heart: very nice to have references in code docstrings ! and at click-only distances through URLs !!!